backend/cms/views/statistics/statistics.py

```diff
@@ -40,23 +44,29 @@ def prepare_csv(languages, hits, dates):
         return csv_raw
 
     def get(self, request, *args, **kwargs):
+        site_slug = kwargs.get('site_slug')
+        site = Site.objects.get(slug=site_slug)
         start_date = request.GET.get('start_date', str(date.today() -
                                                        timedelta(days=30)))
         end_date = request.GET.get('end_date', str(date.today()))
         languages = [["de", "Deutsch", "#7e1e9c"], ["en", "Englisch", "#15b01a"],
                      ["ar", "Arabisch", "#0343df"]]
 
-        api_man = MatomoApiManager(matomo_url="https://statistics.integreat-app.de",
-                                   matomo_api_key="",
+        api_man = MatomoApiManager(matomo_url=site.matomo_url,
+                                   matomo_api_key=site.matomo_token,
                                    ssl_verify=True)
         response_dates = []
         response_hits = []
         for lang in languages:
-            api_hits = api_man.get_visitors_per_timerange(date_string=start_date + ',' + end_date,
-                                                          site_id="2",
-                                                          period=request.GET.get('peri', 'day'),
-                                                          lang=lang[0])
-            temp_hits = []
+            try:
+                api_hits = api_man.get_visitors_per_timerange(date_string=start_date + ',' + end_date,
+                                                              site_id="2",
+                                                              period=request.GET.get('peri', 'day'),
+                                                              lang=lang[0])
+                temp_hits = []
+            except ConnectionError:
```

Comment:
In addition to `ConnectionError`, I also received `InvalidURL` (if no matomo_url is defined) and `TypeError`(if https://statistics.integreat-app.de is given but the key is wrong or the region is not configured properly). So probably, we should also catch these errors.
However, we also could hide the statistics menu item if the url is not set for a specific region (or `statistics_enabled` is `false`).

Reply:
Extra Try-Catch statements have been added